app/views/bastion/layouts/application.html.erb

```diff
@@ -1,3 +1,7 @@
+<% content_for(:head) do %>
+    <base href="/bastion"/>
```

Comment:
https://github.com/angular-ui/ui-router/wiki/Frequently-Asked-Questions#issue-my-assets-and-templates-are-not-loading

Without it I was getting a javascript error when attempting to go to any new katello page.